### PR TITLE
Add bounce ethereum tvl

### PR DESCRIPTION
**Pull request by yuhuakukude**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Bounce

##### Twitter Link:
https://twitter.com/bounce_finance?s=21

##### Website Link:
https://www.bounce.finance

##### Logo (High resolution, will be shown with rounded borders):
logo

##### Current TVL:
$6.72M

##### Chain:
ethereum

##### Short Description (to be shown on DefiLlama):
From customized auction pools to seamless on-chain settlements, Bounce Finance provides the tools you need to unleash the full potential of decentralized auctions.

##### Token address and ticker if any:
0xa9b1eb5908cfc3cdf91f9b8b3a74108598009096

##### Category (full list at https://defillama.com/categories) *Please choose only one:
[Launchpad](https://defillama.com/protocols/Launchpad)

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
Chainlink

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/orgs/bouncefinance

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/bounce exports TVL: 
        
 
 
    ethereum                  6.32 M
    
    total                    6.32 M

**Comment by llamatester** [comment]
The adapter at projects/bounce exports TVL: 
        
 
 
    ethereum-staking          1.48 M
    ethereum                  7.85 M
    staking                   1.48 M
    
    total                    7.85 M

File: projects/bounce/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const { sumTokensAndLPsSharedOwners } = require('../helper/unwrapLPs')
+
+const AUCTION = '0xA9B1Eb5908CfC3cdf91F9B8B3a74108598009096'
+const AUCTION_ETH_SLP = '0x0f8086d08a69ebd8e3a130a87a3b6a260723976f'
+
+const STAKING_ADDRESS = '0x98945BC69A554F8b129b09aC8AfDc2cc2431c48E'
+const STAKING_LP_ADDRESS = '0xbe5a88b573290e548759520a083a61051b258451'
+const MULTI_SIG_ADDRESS = '0xc9297466C6c7acc799Fb869806C53398b8B10680'
```
Comment (g1nt0ki) [inline projects/bounce/index.js:8]: this is your project treasury right? why are you counting it as tvl?

I have refactored the code here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7109/files

**Comment by g1nt0ki** [comment]
closing in favor of https://github.com/DefiLlama/DefiLlama-Adapters/pull/7109

**Pull request closed without merging**